tests/main/debug-execution/task.yaml

```diff
@@ -8,6 +8,11 @@ debug: |
     grep -n '' snap-*.out || true
 
 execute: |
+    # TODO: remove this check once snapd 2.65 is released
```

Comment:
I think this todo cannot be removed because we will always have a distribution with old snapd.

I think the test needs more nuance to execute the SNAP_REEXEC=1 without other conditions.